### ISSUE TITLE
[Bug]: Bars-tools component accessibility issues

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ods-ios#580]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Plus

### OS Version

17.1.1

### App version

v0.16.0
### Describe the issue

This list is about "Bars-tools" component and variant only. Other common issues are described in dedicated issues.

- [ ] #723
**Recommandation: Items type should be usable with keyboard**
![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/d52bb1dc-c2e4-4348-9a6b-8b92997d0df4)

- [x] Keyboard focus is not visible when reaching the "Open sheet" button
**Recommandation: focus should be visible on "Open sheet" button**

### Expected Behavior
Page 100% accessible
